server/internal/server/org.go

```diff
@@ -109,17 +109,44 @@ func (s *S) ListOrganizations(ctx context.Context, req *v1.ListOrganizationsRequ
 	}
 
 	// Only show orgs that the user is a owner/reader of.
-	var filtered []*store.Organization
+	var visibleOrgs []*store.Organization
 	for _, org := range orgs {
 		if s.organizationRole(org.OrganizationID, userInfo.UserID) != v1.OrganizationRole_ORGANIZATION_ROLE_UNSPECIFIED {
-			filtered = append(filtered, org)
+			visibleOrgs = append(visibleOrgs, org)
 		}
 	}
 
 	var orgProtos []*v1.Organization
-	for _, org := range filtered {
+	for _, org := range visibleOrgs {
 		orgProtos = append(orgProtos, org.ToProto())
 	}
+
+	if req.IncludeSummaries {
+		for _, org := range orgProtos {
+			ps, err := s.store.ListProjectsByTenantIDAndOrganizationID(userInfo.TenantID, org.Id)
+			if err != nil {
+				return nil, status.Errorf(codes.Internal, "list projects: %s", err)
+			}
+
+			var visiblePjCount int32
+			for _, p := range ps {
+				if s.validateProjectMember(p.ProjectID, p.OrganizationID, userInfo.UserID) == nil {
+					visiblePjCount++
+				}
+			}
+
+			us, err := s.store.ListOrganizationUsersByOrganizationID(org.Id)
```

Comment:
Can't we also call `validateOrganizationOwner` and show the user count only if the return value is `nil`? This follows https://github.com/llmariner/user-manager/blob/main/server/internal/server/org.go#L144

Reply:
done!